spyder/utils/conda.py

```diff
@@ -92,3 +101,45 @@ def get_conda_env_path(pyexec, quote=False):
         conda_env = add_quotes(conda_env)
 
     return conda_env
+
+
+def get_list_conda_envs():
+    """Return the list of all conda envs found in the system."""
+    conda_list = ['conda', 'env', 'list', '--json']
+    if sys.platform == 'darwin':
+        # This is needed when Spyder is run from Spyder.app
+        conda_path = find_program('conda')
+        if not conda_path:
+            conda_list = [get_home_dir(), 'opt', 'anaconda3', 'condabin',
+                          'conda', 'env', 'list', '--json']

```

Comment:
Thanks @mrclary for this! On Friday I tried unsuccessfully, and then we decided that we will take care of this problem for the mac installer later. I'll add this to the PR so that will work :)!